.github/workflows/awaiting-response.yml

```diff
@@ -0,0 +1,20 @@
+name: awaiting response
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '5 * * * *'
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
+        with:
+          closeComment: >
```

Comment:
The wording feels a bit weird. How about something more like this? (Feel free to put the "Of course," back in, I don't feel that strongly about it)
> This issue has been automatically closed because it has been blocked for too long on a response that wasn't provided. If the needed response is provided later, re-open this ticket so that we can progress it further. :)